src/Compilers/CSharp/Portable/Binder/BuckStopsHereBinder.cs

```diff
@@ -42,6 +42,8 @@ protected override LocalFunctionSymbol LookupLocalFunction(SyntaxToken nameToken
             return null;
         }
 
+        internal override uint LocalScopeDepth => 0;
```

Comment:
Use constant?

Reply:
Yes, that would be better.

---
In reply to: [135400648](https://github.com/dotnet/roslyn/pull/21334#discussion_r135400648) [](ancestors = 135400648)